bindings/python/pmix.pyx

```diff
@@ -251,6 +251,68 @@ cdef class PMIxClient:
             pmix_free_info(info, ninfo)
         return rc
 
+    # retrieve a value from  the keystore
+    #
+    # @proc [INPUT]
+    #       - namespace and rank of the client (tuple)
+    #
+    # @ninfo [INPUT]
+    #      - number of entries in dicts list
+    #
+    # @key [INPUT]
+    #      - the key to be retrieved
+    #
+    # @dicts [INPUT]
+    #          - a list of dictionaries, where each
+    #            dictionary has a key, value, and val_type
+    #            defined as such:
+    #            [{key:y, value:val, val_type:ty}, … ]
+    #
+    # @value [INPUT]
+    #        - value to be retrieved from keystore
+    def get(self, proc:tuple, ky, dicts:list, num_info):
```

Comment:
Sorry - I should have caught this earlier. No need to pass `num_info` as it is simply the number of keys in `dicts`.

Reply:
@rhc54 ok, I will remove. I guess I was originally just following the PMIx_Get signature, but I see now I don't need to do that.

Reply:
@rhc54 removed it, thanks

Reply:
Thanks - we want the Python bindings to be as "native" as possible, not just mirrors of the C bindings.